fedimint-bitcoind/src/bitcoincore.rs

```diff
@@ -0,0 +1,104 @@
+use anyhow::anyhow as format_err;
+use bitcoin::{BlockHash, Network, Transaction, Txid};
+use bitcoincore_rpc::bitcoincore_rpc_json::EstimateMode;
+use bitcoincore_rpc::{Auth, RpcApi};
+use fedimint_core::task::{block_in_place, TaskHandle};
+use fedimint_core::{apply, async_trait_maybe_send, Feerate};
+use tracing::info;
+use url::Url;
+
+use crate::{DynBitcoindRpc, IBitcoindRpc, IBitcoindRpcFactory, RetryClient};
+
+#[derive(Debug)]
+pub struct BitcoindFactory;
+
+impl IBitcoindRpcFactory for BitcoindFactory {
```

Comment:
Oh dear. Do we really need a whole factory?

Reply:
That has been there for a while now (just got moved around), and I think it's needed for dynamic BTC backend choice at runtime.

Reply:
Yeah since we need to create multiple clients in places.  This didn't really change but moved, I'll add a doc.